src/lib/Microsoft.Fx.Portability.Reports.Html/Views/_PortabilityReport.cshtml

```diff
@@ -142,12 +142,13 @@
                 {
                     var name = Model.RemoveTypeOrMemberPrefix(type.TypeName);
                     <tr>
-                        <td>@name</td>
+                        @*Make type name row to have different background color with member row for nicer looking*@
+                        <td style="background:#E7E7E8">@name</td>
                         @foreach (var version in Model.GetMatchingTargetsAndSupportedVersions(type.TargetVersionStatus))
                         {
-                            @Html.TargetSupportCell(version)
+                            @Html.TargetSupportCell(version, "background:#E7E7E8");
                         }
-                        <td>@type.RecommendedChanges</td>
+                        <td style="background:#E7E7E8">@type.RecommendedChanges</td>
```

Comment:
Why not do this in the style sheet? inline styles like this lead to difficult-to-maintain styling.